Doc 1776/open banking partner documentation on docsswanio (#641)
swan-io/docs: docs/topics/definitions/_consent-token.mdx

A 180-day token granting a Third-Party Provider Open Banking access to a user's Swan account.

> The consent token is issued after the user completes Strong Customer Authentication and must be renewed every 180 days, as required by PSD2.

Reviewer: ```suggestion
Swan issues the consent token once the user completes Strong Customer Authentication and must be renewed every 180 days, as required by PSD2.
```
